railties/lib/rails/commands/dbconsole/dbconsole_command.rb

```diff
@@ -167,7 +167,7 @@ module Command
     class DbconsoleCommand < Base # :nodoc:
       include EnvironmentArgument
 
-      class_option :include_password, aliases: "-p", type: :boolean,
+      class_option :include_password, aliases: "-p", type: :boolean, default: true,
         desc: "Automatically provide the password from database.yml"
```

Comment:
If we're setting the default value of a boolean param to true, would it even be possible for it to ever be false? Seems like the param should be deprecated/removed if it isn't going to have an effect any more.

This will also definitely need a changelog entry

Reply:
Thanks for looking at this PR! I really appreciate it.

The dbconsole command documents flags to both enable/disable this option:

```
Usage:
  rails dbconsole [options]

Options:
  -e, [--environment=ENVIRONMENT]                    # Specifies the environment to run this dbconsole under (test/development/production).
  -p, [--include-password], [--no-include-password]  # Automatically provide the password from database.yml
                                                     # Default: true
      [--mode=MODE]                                  # Automatically put the sqlite3 database in the specified mode (html, list, line, column).
                                                     # Possible values: html, list, line, column
      [--header], [--no-header]
  --db, [--database=DATABASE]                        # Specifies the database to use.
  ```

I believe boolean options provide a command line flag to both enable or disable the value.  Currently, this option is disabled, but running `bin/rails dbconsole --include-password` will enable it.

With this PR changing the default value to true, it can still be disabled if users specify the `bin/rails dbconsole --no-include-password` flag on the command.  I've just confirmed this flag will still disable the value even with a default of true.

Reply:
I've added a `railties` CHANGELOG entry in b98e8918749b424f95552dbb6f11a6d481609636 describing this behavior change.